core/setup.py

```diff
@@ -30,7 +32,19 @@ def main():
             [console_scripts]
             autocrypt=autocrypt.cmdline:autocrypt_main
         ''',
-        install_requires = ["click>=6.0", "six"],
+        install_requires=[
+            "click>=6.0",
+             "six"
+        ],
+        dependency_links=[
+            "https://github.com/SecurityInnovation/PGPy.git@\
+             @release/0.4.1#egg=PGPy-0.4.1"
```

Comment:
this also needs to be a normal install_requires pinned release -- unless this is not a released version of pgpy in which case we need to wait until they release it. (dependency_links does not work in some pip versions and requires extra options for "pip install" ASFAIU and no project i am involved in uses it).